cross-chain/base/deploy_l2/22_transfer_base_wormhole_gateway_ownership.ts

```diff
@@ -0,0 +1,19 @@
+import type { HardhatRuntimeEnvironment } from "hardhat/types"
+import type { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { getNamedAccounts, helpers } = hre
+  const { deployer, governance } = await getNamedAccounts()
+
+  await helpers.ownable.transferOwnership(
+    "BaseWormholeGateway",
+    governance,
+    deployer
+  )
+}
+
+export default func
+
+func.tags = ["TransferBaseWormholeGatewayOwnership"]
+func.dependencies = ["BaseWormholeGateway"]
```

Comment:
Do we want to add as dependencies all the scripts that update `BaseWormholeGateway`?
```suggestion
func.dependencies = ["BaseWormholeGateway", "SetBaseGatewayAddress", "SetArbitrumGatewayAddress", "SetPolygonGatewayAddress", "SetOptimismGatewayAddress"]
```

Reply:
I don't think we need them here
- we're going to split the deployment part into 2 phases on mainnet. The scripts that transfer ownerships will be in the 2 phase and we will need to comment out this dependency line anyway
- the second reason is that all the Set* dependencies are the setting of the gateways, and technically it is not required to set them before we transfer the ownership. It's just some convenience for devs (and we will set the gateways before the ownership transfer anyway). See `BaseWormholeGateway` it is required to have it for the ownership transfer, but we don't need to set anything for the ownership transfer to work.